docs/publish-module/publish-information/resign-binary.md

```diff
@@ -0,0 +1,108 @@
+
+- **Field**: `CFBundleShortVersionString`
+- **Description**: The release or version number of the bundle.
+
+#### Build Number
+
+- **Field**: `CFBundleVersion`
+- **Description**: The version of the build that identifies an iteration of the bundle.
+
+#### Entitlements
+
+- **Options**:
+  - **Entitlements from provisioning profiles**: Use entitlements from the new provisioning profile.
+  - **Combine app entitlements**: Extract app bundle code signing entitlements and combine with entitlements from new provisioning profiles.
+  - **New app entitlements**: Edit this XML file to edit capabilities. Resigning process uses a single entitlement XML file.
+
+#### Targets
+
+- **Description**: Select a new provisioning profile for new targets.
+
+#### Bundle ID
+
+- **Field**: `CFBundleIdentifier`
+- **Description**: Original Bundle ID.
+
+#### Provisioning Profiles
+
+- **Description**: Select a new provisioning profile for the Bundle ID.
+
+### Resigning Process
+
+To resign a binary, follow these steps:
+
+1. **Select the Version**: Choose the version of your app you wish to resign from the **Version List** in the Publish module.
+2. **Configure Resigning Options**: Navigate to the **Resign Binary** action and configure the necessary fields such as the provisioning profile, entitlements, and other settings.
+3. **Sign the Binary**: After configuring, click the **Sign** button to resign the binary. This process will create a new package with the updated provisioning profile and entitlements.
+
+### Post-Resignation
+
+Once the binary is resigned, a new package is automatically created to reflect the changes. This ensures that any distribution or testing utilizes the most current setup without requiring a complete rebuild.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/be-3161-publish-resigned.png' />
+
+The newly created package can then be distributed or tested according to your workflow requirements. This update ensures that your application conforms to the necessary provisioning and entitlement specifications for different environments, such as development, staging, or production, without additional build steps.
+
+This feature streamlines the application update process by allowing for quick adjustments to the app's configurations, significantly reducing the time and resources needed for separate build cycles.
+
+## Resign Android Binary
+
+Resigning an Android binary allows you to apply a new keystore to your application after the initial build. This is useful for updating the signing configuration or switching to a different keystore as needed without needing to rebuild the app.
+
+### Fields and Options
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/be-3161-publish-resign-android-option.png' />
+
+- **Package ID**: This is the unique identifier for your Android application, also known as the application ID. It usually follows the format `com.example.myapp` and should not be changed during the resigning process.
+
```

Comment:
:::caution Package ID

Please note that **Package ID** changes are not allowed while the related version is resigned in the **Publish module**. If you want to change the **Package ID value** of your package, please use the **Resign Binary** feature in [**Testing Distribution**](/distribute/platform-specific-guidance/android/resigning-android-binaries).

:::